_posts/2019-06-16-release-1.8.1.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title:  "Apache Flink 1.8.1 Released"
+date:   2019-06-16 12:00:00
+categories: news
+authors:
+- till:
+  name: "Jincheng Sun"
+---
+
+The Apache Flink community released the first bugfix version of the Apache Flink 1.8 series.
+
+This release includes more than 40 fixes and minor improvements for Flink 1.8.1, The list below includes a detailed list of all fixes.
```

Comment:
I would update this with: 

This release includes more than 40 fixes and minor improvements for Flink 1.8.1. Below we include a detailed list of all improvements, sub-task and bug fixes in the latest release.

Reply:
I like add `all improvements, sub-tasks and bug fixes`. :)